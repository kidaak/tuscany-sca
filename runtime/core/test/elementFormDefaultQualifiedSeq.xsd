<?xml version="1.0" encoding="UTF-8"?>
<!--
   Licensed to the Apache Software Foundation (ASF) under one
   or more contributor license agreements.  See the NOTICE file
   distributed with this work for additional information
   regarding copyright ownership.  The ASF licenses this file
   to you under the Apache License, Version 2.0 (the
   "License"); you may not use this file except in compliance
   with the License.  You may obtain a copy of the License at
   
     http://www.apache.org/licenses/LICENSE-2.0
     
   Unless required by applicable law or agreed to in writing,
   software distributed under the License is distributed on an
   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
   KIND, either express or implied.  See the License for the
   specific language governing permissions and limitations
   under the License.
-->

<xs:schema targetNamespace="test"
  xmlns="test" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">

	<xs:element name="x" type="xT"/>
	
	<xs:complexType name="xT">
		<xs:choice maxOccurs="unbounded">
			<xs:element name="a" type="aT" minOccurs="0" maxOccurs="unbounded" />
			<xs:element name="b"  type="xs:string"  />
 		</xs:choice>
	</xs:complexType>
	
	<xs:complexType name="aT">
		<xs:choice maxOccurs="unbounded">
			<xs:element name="a1" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
            <xs:element name="a2" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
		</xs:choice>
	</xs:complexType>
	
</xs:schema>