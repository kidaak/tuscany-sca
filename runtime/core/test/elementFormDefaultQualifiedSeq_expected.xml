<?xml version="1.0" encoding="UTF-8"?>
<a xsi:type="tns2:aT" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:tns2="test">
  <tns2:a1>test data</tns2:a1>
</a>
